skimage/measure/_marching_cubes_lewiner.py

```diff
@@ -119,12 +126,14 @@ def marching_cubes(volume, level=None, *, spacing=(1., 1., 1.),
     if method == 'lewiner':
         return _marching_cubes_lewiner(volume, level, spacing,
                                        gradient_direction, step_size,
-                                       allow_degenerate, use_classic=False)
+                                       allow_degenerate, use_classic=False, mask=mask)
     elif method == 'lorensen':
         return _marching_cubes_lewiner(volume, level, spacing,
                                        gradient_direction, step_size,
-                                       allow_degenerate, use_classic=True)
+                                       allow_degenerate, use_classic=True, mask=mask)
     elif method == '_lorensen':
+        if mask is not None:
+            warnings.warn('Attribute mask is not implemented for _lorensen method.')
```

Comment:
```suggestion
            warnings.warn(
                'Parameter `mask` is not implemented for method "_lorensen" '
                'and will be ignored.', stacklevel=2
            )
```
This ensures that the warning points the user at the place where the function `marching_cubes` is called and not at its internals. I also took the liberty to tweak the wording a little bit because "Attribute" as a different meaning in the Python space.

Reply:
oh of course I mixed it with argument but yes, parameter also works. Thank you very much